### DISCUSSION TITLE
Is it possible to add the moonpay option into appkit's buy crypto section?

### DISCUSSION THREAD

**Discussion by Mewsxd** [reown-com/appkit#3532] [Q&A]
I'm building an app and i need to add moonpay as one of the options for the users to buy crypto, so i was wondering if it would be possible to add this to reown. Any help would be appreciated

**Comment by arein** [comment]
@Mewsxd we have a direct integration Coinbase Pay and with Meld which is an onramp aggregator. We're currently adding more and more options through Meld. That's our strategy of expanding to more and more onramp providers.

If you still need Moonpay you'd need to manually add that yourself into your app. We have no short-term plans of adding Moonpay.

May I ask what you are building?

**Reply by Mewsxd** [reply]
Thank you for your quick reply, I have integrated moonpay into my app outside of the reown appkit. I'm building an app for a new coin launch where a user could swap their native token of the chain for this new coin
